app/javascript/controllers/pathogen/datepicker/constants.js

```diff
@@ -4,6 +4,23 @@ export const FOCUSABLE_ELEMENTS =
 
 export const DAYS_IN_MONTH = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
 
+// specifically required for when utils.js getDayOfWeek() is called as new Date(date).getDay() doesn't work with
```

Comment:
1) Remove this.  Instead, in `app/javascript/controllers/pathogen/datepicker/utils.js` update the `getDayofWeek` function to:

```javascript
export function getDayOfWeek(year, monthIndex, day) {
  return new Date(year, monthIndex, day).getDay();
}
```